app/controllers/api/v1/work_groups_controller.rb

```diff
@@ -65,8 +65,10 @@ def set_and_authorize_work_group
   end
 
   def work_group_params
+    # params.require(:work_group).permit(:name, :classroom_id, :session_time, :aasm_state,
```

Comment:
Noticed that there are two session_time, maybe one was supposed to be aasm_state? I know it may not be a big deal but for the sake of code readability, I was wondering if it should be changed.